gsd/hoomd.py

```diff
@@ -1092,3 +1097,58 @@ def open(name, mode='rb'):
                          schema_version=[1, 4])
 
     return HOOMDTrajectory(gsdfileobj)
+
+def read_log(name, scalar_only=True):
+    """Read scalar values from a hoomd schema GSD file.
+
+    Args:
+        name (str): File name to open.
+
+    Returns:
+        `pandas.DataFrame`
+    """
+    if fl is None:
+        raise RuntimeError("file layer module is not available")
+    if gsd is None:
+        raise RuntimeError("gsd module is not available")
+    if pandas is None:
+        raise RuntimeError("pandas module is not available")
+
+    gsdfileobj = fl.open(name=str(name),
+                         mode='rb',
+                         application='gsd.hoomd ' + gsd.__version__,
+                         schema='hoomd',
+                         schema_version=[1, 4])
+
+    logged_data_names = gsdfileobj.find_matching_chunk_names('log/')
+    # Always log timestep associated with each log entry
+    logged_data_names.insert(0, 'configuration/step')
+    if len(logged_data_names) > 1:
+        logged_data_dict = dict()
+        for log in logged_data_names:
+            if gsdfileobj.chunk_exists(frame=0, name=log):
+                tmp = gsdfileobj.read_chunk(frame=0, name=log)
+                if scalar_only and not tmp.shape[0] == 1:
+                    pass
+                else:
+                    logged_data_dict[log] = numpy.zeros_like(tmp, shape=(gsdfileobj.nframes, *tmp.shape))
+                    logged_data_dict[log][0] = tmp
+
+        for idx in range(1, gsdfileobj.nframes):
+            for log in logged_data_dict.keys():
+                if not gsdfileobj.chunk_exists(frame=idx, name=log):
+                    idx = 0
+                logged_data_dict[log][idx] = gsdfileobj.read_chunk(frame=idx, name=log)
+
+        return logged_data_dict
+    else:
+        raise RuntimeError('No logged data in file: ' + name)
+
+def read_log_as_dataframe(name, **kwargs):
```

Comment:
Just document that
```python
df = pd.DataFrame(read_log(name, scalar_only=True))
```
will get a dataframe with the logged data, and remove the function.